tests/Fixer/Operator/OperatorLinebreakFixerTest.php

```diff
@@ -0,0 +1,207 @@
+     * @param string      $expected
+     * @param null|string $input
+     * @param array       $configuration
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFixControlContinuationBraces($expected, $input = null, array $configuration = [])
+    {
+        $this->fixer->configure($configuration);
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        yield [
+            '<?php
+$foo
+    = $bar;
+',
+            '<?php
+$foo =
+    $bar;
+',
+        ];
+        yield [
+            '<?php
+return $foo
+    + $bar;
+',
+            '<?php
+return $foo +
+    $bar;
+',
+        ];
+
+        yield [
+            '<?php
+return $foo +
+    $bar;
+',
+            null,
+            ['only_booleans' => true],
+        ];
+
+        yield [
+            '<?php
+return $foo
+    || $bar
+    || $baz;
+',
+            '<?php
+return $foo ||
+    $bar ||
+    $baz;
+',
+        ];
+
+        yield [
+            '<?php
+return $foo
+    || $bar;
+',
+            '<?php
+return $foo
+    ||
+    $bar;
+',
+        ];
+
+        yield [
+            '<?php
+return $foo ||
+    $bar ||
+    $baz;
+',
+            '<?php
+return $foo
+    || $bar
+    || $baz;
+',
+            ['position' => 'end'],
+        ];
+
+        yield [
+            '<?php
+return $foo ||
+    $bar;
+',
+            '<?php
+return $foo
+    ||
+    $bar;
+',
+            ['position' => 'end'],
+        ];
+
+        yield [
+            '<?php
+function foo() {
+    return $a
+        || $b;
+}
+',
+            '<?php
+function foo() {
+    return $a||
+        $b;
+}
+',
+        ];
+
+        yield [
+            '<?php
+function foo() {
+    return $a ||
+        $b;
+}
+',
+            '<?php
+function foo() {
+    return $a
+        ||$b;
+}
+',
+            ['position' => 'end'],
+        ];
+
+        yield [
+            '<?php
+function getNewCuyamaTotal() {
+    return 562 // Population
+        + 2150 // Ft. above sea level
+        + 1951; // Established
+}
+',
+            '<?php
+function getNewCuyamaTotal() {
+    return 562 + // Population
+        2150 + // Ft. above sea level
+        1951; // Established
+}
+',
+        ];
+
+        yield [
+            '<?php
+function getNewCuyamaTotal() {
+    return 562 /* Population */
+        + 2150 /* Ft. above sea level */
+        + 1951; /* Established */
+}
+',
+            '<?php
+function getNewCuyamaTotal() {
+    return 562 + /* Population */
+        2150 + /* Ft. above sea level */
+        1951; /* Established */
+}
+',
+        ];
+
+        yield [
+            '<?php
+function foo() {
+    return isThisTheRealLife() // First comment
+        // Second comment
+        // Third comment
+        || isThisJustFantasy();
+}
+',
+            '<?php
+function foo() {
+    return isThisTheRealLife() || // First comment
+        // Second comment
+        // Third comment
+        isThisJustFantasy();
+}
+',
+        ];
+    }
```

Comment:
I'd like to see some nested testcases, you know, something like

```
return $a
    && (
        $b
        || $c
    )
    && $d;
```

(And then some variations, with comments too.)